config/settings/base.py

```diff
@@ -355,6 +355,7 @@
 MAILJET_MASTER_API_SECRET = env.str("MAILJET_MASTER_API_SECRET", "")
 # ID of buyers contact list
 MAILJET_NEWSLETTER_CONTACT_LIST_BUYER_ID = env.int("MAILJET_NEWSLETTER_CONTACT_LIST_BUYER_ID", 2500034)
+MAILJET_NEWSLETTER_CONTACT_LIST_SELLER_ID = env.int("MAILJET_NEWSLETTER_CONTACT_LIST_SELLER_ID", 2500034)
```

Comment:
```suggestion
MAILJET_NEWSLETTER_CONTACT_LIST_SIAE_ID = env.int("MAILJET_NEWSLETTER_CONTACT_LIST_SIAE_ID", 2500034)
```

Dans l'app on utilise `SIAE` pour parler des utilisateurs "structures"
https://github.com/betagouv/itou-marche/blob/631a86718762334df7abe1b1bee2870d0cca648e/lemarche/users/models.py#L91